### PR TITLE
Remove 0x3e3x did from cometlake

### PR DESCRIPTION
**Pull request by jun2zhou**

### CODE REVIEW

File: chipsec/cfg/8086/pch_3xxop.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@ https://www.intel.com/content/www/us/en/products/docs/chipsets/300-series-chipse
   <!-- #################################### -->
   <info>
     <sku did="0x9D84" name="PCH-U" code="PCH_3xxOP" longname="Intel 300 series On-Package PCH" />
+    <sku did="0x9D85" name="PCH-U" code="PCH_3xxOP" longname="Intel 300 series On-Package PCH" />
```
Comment (npmitche) [inline chipsec/cfg/8086/pch_3xxop.xml:16]: This change should not be in this PR as it's a part of #1784

File: chipsec/cfg/8086/cml.xml [hunk 2]
```diff
@@ -8,10 +8,6 @@
   <!--                                      -->
   <!-- #################################### -->
   <info family="core" detection_value="A0652, A0653, A0655, A0660, A0661">
-    <sku did="0x3E20" name="CometLake" code="CML" longname="CometLake H8 Core / XeonW" />
```
Comment (npmitche) [inline chipsec/cfg/8086/cml.xml:11]: What's the reason for removing these DIDs?

**Review by npmitche — CHANGES_REQUESTED** [review]

File: chipsec/cfg/8086/cml.xml (hunk 2 above)
Comment (jun2zhou) [reply chipsec/cfg/8086/cml.xml:11]: Comet Lake and Whiskey Lake CPUID and DID collision, refer CHIPSEC-803

File: chipsec/cfg/8086/pch_3xxop.xml (hunk 1 above)
Comment (jun2zhou) [reply chipsec/cfg/8086/pch_3xxop.xml:16]: will remove

**Comment by jun2zhou** [comment]
will re-submit

**Pull request closed without merging**